lib/sales_analyst.rb

```diff
@@ -60,4 +59,27 @@ def average_average_price_per_merchant
     BigDecimal.new(array.reduce(:+) / array.count).round(2)
   end
 
+  def find_standard_deviation_of_average_item_price
+    amount_of_items = @se.items.all.count
+    average_price_of_items = self.average_average_price_per_merchant
+    prices = []
+    @se.items.all.each do |item|
+      prices << ((item.price - average_price_of_items)**2)
+    end
+    summed_prices = (prices.reduce(:+)/(amount_of_items - 1))
+    Math.sqrt(summed_prices).round(2)
+  end
+
+  def golden_items
```

Comment:
Split Golden Items method into two methods, one for standard deviation calculation and the other for golden item.